src/components/ftbot/PairlistConfigurator.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <b-container fluid>
+    <b-row align-v="stretch">
+      <b-col cols="12" md="3" style="height: 760px; overflow-y: scroll">
+        <div ref="availablePairlistsEl" class="available-pairlists">
+          <b-row
+            v-for="pairlist in availablePairlists"
+            :key="pairlist.name"
+            align-v="center"
+            class="pairlist"
+          >
+            <b-col>
+              {{ pairlist.name }}
```

Comment:
i think we should show have the description here, too.
there's plenty of space if we remove the extensive vertical margins (make the individual fields smaller).

Reply:
Agreed!